buildbuddy.yaml

```diff
@@ -25,6 +25,8 @@ actions:
         test --config=mac-workflows
         --
         //...
+        -//enterprise/server/raft/store:all
```

Comment:
Why are we just excluding the flaky tests for mac?

Reply:
These tests were previously excluded and recently re-enabled; I am just re-disabling them. We can probably increase the timeouts or re-enable them at some point, but I figured it's not worth thinking about right now.

Reply:
Actually, I didn't realize I hadn't submitted https://github.com/buildbuddy-io/buildbuddy/pull/3632 . Let's merge that one first and see if it helps

Reply:
It didn't help - reopening.

Reply:
It still says it's closed for me so I can't approve it